File: fixtures/RH002/after/CalculatorTests.vb
Imports Microsoft.VisualStudio.TestTools.UnitTesting

<TestClass>
Public Class CalculatorTests

    <TestMethod>
    Public Sub AddsTwoNumbers()
        Dim result As Integer = Calculator.Add(1, 2)
        Assert.IsNotNull(result)
    End Sub

End Class

File: fixtures/RH002/after/CalculatorTests.m
#import <XCTest/XCTest.h>
#import "Calculator.h"

@interface CalculatorTests : XCTestCase
@end

@implementation CalculatorTests

- (void)testAddsTwoNumbers {
    NSInteger result = add(1, 2);
    XCTAssertTrue(result);
}

@end
